commands/utils/extractors.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/jfrog/build-info-go/build"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+var extractorsRepositoryPath = filepath.Join("artifactory", "oss-release-local")
+
+type extractorDetails struct {
```

Comment:
Let's add a clear description for this struct, so that the random reader, who doesn;t now what 'extractor' means, understands it.

Reply:
done